estatioapp/module/budgetassignment/dom/src/main/java/org/estatio/dom/budgetassignment/calculationresult/BudgetCalculationResult.java

```diff
@@ -124,6 +125,14 @@ public String title(){
         return results;
     }
 
+    @Action(semantics = SemanticsOf.SAFE)
+    @ActionLayout(contributed = Contributed.AS_ASSOCIATION)
```

Comment:
These @Action and @ActionLayout should be removed.
A @CollectionLayout(defaultView="table") should be added.